### ISSUE TITLE
example server stays at "connecting..."

### ISSUE THREAD

**Issue by hatpick** [share/share-codemirror#4]
Hey,
I get this error when I try to run the server:
Could not subscribe: [object Object]

I'm a ShareJS 0.6.X user and I don't have a good understanding of 0.7+ at the moment so I can't figure out what's wrong, would you mind checking this?

**Comment by aslakhellesoy** [comment]
Did you clone the repo and follow the instructions in the README?

```
npm install
node examples/server.js
# in a couple of browsers...
open http://localhost:7007
```

**Comment by hatpick** [comment]
Yes, that's what I did, I have the server running on 7007, however in the chrome console I can see that error `Could not subscribe: [object Object]`

**Comment by aslakhellesoy** [comment]
That's odd - I'm not able to reproduce this...

**Comment by dignifiedquire** [comment]
@hatpick what version of node are you running?

**Comment by hatpick** [comment]
stable version v0.10.29

btw I can run ShareJS server on the same machine and it works. (from sharejs project)

**Comment by dignifiedquire** [comment]
Reproduced myself, latest master and same node version. Something seems to go wrong in the subscription, not sure what though, the error is on `share.uncompressed.js#940`, the message looks like this:

``` js
{
  "a": "sub",
  "error": {},
  "c": "users",
  "d": "sephx"
}
```

**Comment by hatpick** [comment]
I moved my `server.js` and `index.html` to `node_modules\share` and updated the refs and it works.

**Comment by jsierles** [comment]
FYI, this still happens on master.

**Comment by christopherdro** [comment]
Looks like `share.uncompressed.js` is missing from the main directory and is being called from index.html.

Go to  `/node_modules/share/dist` and copy either the minified or regular js file over to the root directory and reference correctly in index.html

This should get the demo working.

**Comment by nmai** [comment]
@christopherdro Thanks, that worked for me but I wasn't happy with using a workaround. When I went into the code to figure out the actual problem, I realized that the scripts in `share/dist` were still being hosted but they must have been renamed at some point by a ShareJS dev.

All it takes to fix this is renaming `share.uncompressed.js` to `share.js` in `example/index.html`. I'll submit a PR in a bit.
